app/controllers/events_controller.rb

```diff
@@ -13,12 +14,29 @@ class EventsController < ApplicationController
   def index
     @events = Pro::Event.includes(:locations, :network).
               order('-end_event').
+              # Uncomment to restrict to events tagged "culturelover"
+              # where(filters).
+              # Uncomment to restrict to current and future events
+              # where(end_event: ">=" + Date.today.strftime).
```

Comment:
this will need to be conditional once we use it, to take into account looking up past events
`where(end_event: "<" + Date.today.strftime).` < if that's the reverse query it's probably enough to add a ternary around the less/greater-or-equal operator.